dll/win32/newdev/lang/de-DE.rc

```diff
@@ -93,4 +93,7 @@ BEGIN
     IDS_CHSOURCE_TITLE "Wählen Sie Such- und Installationsoptionen"
     IDS_SEARCHDRV_TITLE "Suche..."
     IDS_INSTALLDRV_TITLE "Installiere..."
+
+    /* For IDD_BROWSE_FOR_FOLDER */
+    IDS_BROWSE_FOR_FOLDER_TITLE "Select the folder containing proper drivers for this hardware."
```

Comment:
"Wählen Sie den Ordner aus, der den korrekten Treiber für dieses Gerät enthält."
is the de-DE Translation.

Reply:
@JoachimHenze I'm going to revert all translations that do not belong to PR author anyway before merge, please do this as separate PR / commit once it's merged.